enterprise/models.py

```diff
@@ -118,6 +118,13 @@ class Meta:
         )
     )
 
+    contact_email = models.EmailField(
```

Comment:
I can see more contact info being needed in the future, yes -- for example the phone number of the HR/Learning office, and the name of the training manager.  But maybe this could also be a `contact_user_id` FK field that points to an actual learner record in the system -- I think I'd be alright with that.